src/pages/library/DataBrowserDetails.js

```diff
@@ -48,7 +48,7 @@ const MetadataDetailsComponent = ({ dataObj, name }) => {
       ]),
       div({ style: styles.attributesColumn }, [
         h3({ style: styles.headers }, ['Last Updated']),
-        Utils.makeStandardDate(dataObj['dct:modified'])
+        dataObj.lastUpdated
```

Comment:
Yeah, although interestingly it didn't seem to actually work here, so I am moving to `dataObj['dct:modified'] && Utils.makeStandardDate(dataObj['dct:modified'])`

I think maybe we wait until we have a schema, then go through the UI and make it manage fields based off the schema. Because it's not like we have definitive information right now.

Reply:
(It worked for the null case, but didn't work for the non-null case, which I hadn't checked yet.)

Reply:
@pshapiro4broad I am going to file a ticket to audit the UI for cases like this to do after we are done with the schema, because it would be good to drive the UI off of only schema defined values.